src/io.c

```diff
@@ -837,3 +837,39 @@ add_filter_module(nmsgtool_ctx *c, const char *args) {
 
 	my_free(tmp);
 }
+
+void
+add_stats_module(nmsgtool_ctx *c, const char *args) {
+	char *tmp = NULL;
+	char *saveptr = NULL;
+	char *mod_name = NULL;
+	char *mod_param = NULL;
+	size_t len_mod_param = 0;
+	nmsg_statsmod_t smod;
+
+	/* Parse arguments */
+	tmp = my_strdup(args);
+	mod_name = strtok_r(tmp, ",", &saveptr);
+	mod_param = strtok_r(NULL, "", &saveptr);
+	if (mod_param != NULL) {
+		len_mod_param = strlen(mod_param) + 1;
+	}
+
+	/* Load the stats module. */
+	if (c->debug >= 2)
+		fprintf(stderr, "%s: adding stats module %s\n", argv_program, args);
+
+	smod = nmsg_statsmod_init(mod_name, mod_param, len_mod_param);
+	if (smod == NULL) {
+		if (c->debug >= 2)
```

Comment:
maybe this should always be printed, and not just if debug>=2?

Reply:
I kept this in place for consistency with the filter module plugin behavior (for which the consequences of failure are arguably more serious).